plugin/internal/remotecall/remotecall.go

```diff
@@ -0,0 +1,227 @@
+package remotecall
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+)
+
+const (
+	UserAgentKey        = "User-Agent"
+	UserAgentSSHGateway = "SSH-gateway"
```

Comment:
following other casing conventions here, should it be?
```suggestion
	UserAgentSSHGateway = "SSH-Gateway"
```